src/theme/index.hbs

```diff
@@ -72,13 +78,13 @@
 
                 <!-- Mobile navigation buttons -->
                 {{#previous}}
-                    <a href="{{link}}" class="mobile-nav-chapters previous">
+                    <a rel="prev" href="{{link}}#content" class="mobile-nav-chapters previous">
```

Comment:
In 'eww', #content causes prev/next to jump past the TOC to the content. A "skip to content" link would probably be better for supporting screen readers etc, this is just what I'm using to generate books locally.